gen/calc.py

```diff
@@ -77,6 +77,18 @@ def calculate_mesos_dns_resolvers_str(resolvers):
         return '"externalOn": false'
 
 
+def validate_mesos_log_retention_mb(mesos_log_retention_mb):
+    assert int(mesos_log_retention_mb) >= 512, "Must retain at least 512 MB of logs"
```

Comment:
I'd suggest this actually be set at 1GB / 1024MB. As long as these are considered "mission critical", we want more than 512 MB probably.